### PR TITLE
Create Command_seed.java

### PR DESCRIPTION
**Pull request by Paldiu**

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
```
Comment (VideoGameSmash12) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java:8]: missing permissions annotation, commands without these will cause the whole plugin to shit the bed

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        }
+
+        if (!(sender instanceof Player)) {
+            sender.sendMessage("To check the world seed from console, you must provide a world name!");
+            return true;
+        }
+
+        Player player = (Player) sender;
+        World world = player.getWorld();
+        player.sendMessage(String.valueOf(world.getSeed()));
```
Comment (VideoGameSmash12) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java:30]: why not just make the response look exactly like what it looks like in vanilla?

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            if (world == null) {
+                sender.sendMessage("World " + args[0] +" not found!");
+                return true;
+            }
+            sender.sendMessage(String.valueOf(world.getSeed()));
+            return true;
+        }
+
+        if (!(sender instanceof Player)) {
+            sender.sendMessage("To check the world seed from console, you must provide a world name!");
```
Comment (VideoGameSmash12) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java:24]: this effectively removes functionality from the original command, probably should just get the overworld and use it as a fallback

**Review by VideoGameSmash12 — CHANGES_REQUESTED** [review]
quite a lot of problems

**Comment by Focusvity** [comment]
Closing due to inactivity

**Pull request closed without merging**